seatunnel-apis/seatunnel-api-flink/pom.xml

```diff
@@ -53,6 +53,11 @@
             <groupId>org.apache.flink</groupId>
             <artifactId>flink-streaming-java_${scala.binary.version}</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-optimizer_${scala.binary.version}</artifactId>
```

Comment:
hi @bigdataf why we need this dependency?

Reply:
if we remove `flink-optimizer` dependency, will the flink batch example run successfully?